livekit-agents/livekit/agents/codecs/mp3.py

```diff
@@ -32,8 +35,8 @@ def __init__(self):
                 "You haven't included the decoder_utils optional dependencies. Please install the decoder_utils extra by running `pip install livekit-agents[decoder_utils]`"
             )
         self._closed = False
-        self._input_queue = asyncio.Queue()
-        self._output_queue = asyncio.Queue()
+        self._input_queue = asyncio.Queue[bytes]()
```

Comment:
Let's not type here, bytes and rtc.AudioFrame are wrong since you're also pushing None